README.md

```diff
@@ -1822,6 +1822,8 @@ Retrieve a **collection** of all related topics to a topic.
 
 Add or update a **collection** of all related topics to a topic. This operation is only possible when the server returns the `updateRelatedTopics` flag in the Topic authorization, see section [3.2.8](#328-determining-allowed-topic-modifications)
 
+Throws "409 Conflict" if requestion to add a parent-child relation causing an infinite loop. Example: TopicA is already the parent of TopicB. Making TopicB the parent of TopicA is not allowed, as it will cause an infinite tree-structure.
```

Comment:
```suggestion
Throws "409 Conflict" if requesting to add a parent-child relation causing an infinite loop. Example: TopicA is already the parent of TopicB. Making TopicB the parent of TopicA is not allowed, as it will cause an infinite tree-structure.
```